Interns/SelinAytenCengiz.md

```diff
@@ -0,0 +1,49 @@
+## Hakkında
+
+Ben Selin Ayten Cengiz, Kocaeli Üniversitesi , Bilgisayar Mühendisliği Bölümü öğrencisiyim. 
+
+ C# ,Java ,Kotlin , C  ve Pyhton dilleri hakkında tecrübem var . Unity  , Android ve .Net üzerine eğitim aldım. Daha önce unity ile nişancı web oyunu geliştirdim . Şuan Android ile ilgili eğitimlere ve çalışmalara devam etmekteyim . Eğitim kapsamında küçük çaplı oyunlar ve uygulamalar yapmaya çalışıyorum . Daha önce yaptığım okul projeleri ise daha çok .Net ağırlıklıydı  , birden fazla masaüstü ve web projesi gerçekleştirdim. Şuan için  Android geliştirme üzerine ilerlemek istiyor ve eğitimlere devam ediyorum.
+
+ **Bimser**'de staj yapma fırsatı yakaladım. Şu an Sayın Özgür Tıpırdamaz Bey'in mentörlüğü ile çeşitli projeler ortaya çıkarmaktayız.
+
+## Tarihçe
+
+[**01.08.2022**](https://github.com/bimser-intern/docs/issues/82)
+
+- Android ve IOS Develeoper 'ın iş tanımı ve görevleri hakkında araştırma yapıldı. [Makaleye buradan erişebilirsiniz.](https://www.kariyer.net/pozisyonlar/mobil+uygulama+gelistiricisi/nedir#:~:text=Mobil%20uygulama%20geliştiricisi%2C%20mobil%20cihazlar,kullanıcılara%20hazır%20halde%20sunan%20kişilerdir) 
+
+- Hibrit ve Native mobil uygulama geliştirme hakkında araştırma yapıldı . [Makaleye buradan erişebilirsiniz.](https://bilginc.com/tr/blog/hibrit-karma-vs-native-yerli-mobil-uygulama-gelistirme-257/#:~:text=Tahmin%20edebileceğiniz%20gibi%20en%20çok,ın%20en%20çok%20desteklediği%20dildir)
+
+- Android ve IOS için kullanılan dil ve geliştirme ortamları hakkında araştırma yapıldı .
+
+- Android ve IOS developerlar için roadmap araştırıldı.
+
+-Araştırılan bu Android ve IOS Developer hakkında sunum hazırlandı.
```

Comment:
-'den sonra boşluk bırakılmadığı için markdown listesinin dışında kalmış.
 
![image](https://user-images.githubusercontent.com/36634192/182612068-e4927849-4dc9-4ebf-8c0d-b78a247b6b17.png)

Reply:
Yazım hataları düzeltildi.